webpack/move_to_foreman/common/helpers.js

```diff
@@ -15,3 +17,17 @@ export const KEY_CODES = {
 };
 
 export const getResponseError = ({ data }) => data && (data.displayMessage || data.error);
+
+export const apiError = (actionType, result) => (dispatch) => {
+  const error = getResponseError(result.response);
+  dispatch({
+    type: actionType,
+    result,
+    error,
```

Comment:
Can you move the `result` and `error` to be under `payload` object?
I think we should always dispatch only `type` and `payload` and keep this as a standard.